js/ui/form/components/field_item.js

```diff
@@ -121,7 +121,7 @@ export function renderFieldItem({
         try {
             widgetInstance = createComponentCallback($div, item.editorType, editorOptions);
             widgetInstance.setAria('describedby', helpID);
-            widgetInstance.setAria('labelledby', labelID);
+            if(labelID) widgetInstance.setAria('labelledby', labelID);
```

Comment:
If I understood correctly this line was updated, because if editor options contains a labelledby attr, then form will remove this attr?

Reply:
why it is not this?
```
if(widgetInstance.getAria('labelledby')) {
   widgetInstance.setAria('labelledby', labelID);
}
```

Imho, the low level value should win.